Sources/PostgreSQL/Connection.swift

```diff
@@ -184,7 +196,7 @@ public final class Connection: ConnectionProtocol {
             }
         }
 
-
+		
         guard let result:OpaquePointer = PQexecParams(
```

Comment:
this instantiation makes problems, i.e. line 193:
`parameterData.map { UnsafePointer<Int8>($0) }`. This initialization is no longer available.